From 8372799ff35d53a384393592dab733549bb9d970 Mon Sep 17 00:00:00 2001
From: OpenEmbedded <oe.patch@oe>
Date: Fri, 25 Nov 2022 16:27:30 +0000
Subject: [PATCH] %% original patch: drop-cmake-version.patch

Upstream-Status: Inappropriate [embedded specific]

This is somehow different on Yocto then it is anywhere else
---
 cmake/create_cmake_config_files.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/create_cmake_config_files.cmake b/cmake/create_cmake_config_files.cmake
index 8143f8f..4288c6a 100644
--- a/cmake/create_cmake_config_files.cmake
+++ b/cmake/create_cmake_config_files.cmake
@@ -332,7 +332,8 @@ else()
     # install same cmake configuration file another time into the Modules cmake subdirectory for compatibility reasons
     # We do this only if we did not move yet to exported target, since it does not harmonize
     install(FILES "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
-        DESTINATION share/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules RENAME Find${PROJECT_NAME}.cmake COMPONENT dev)
+  DESTINATION share/cmake/Modules RENAME Find${PROJECT_NAME}.cmake COMPONENT dev)
+
 endif()
 
 # install cmake find_package configuration file
